app/services/recover/create_recover_request.rb

```diff
@@ -1,15 +0,0 @@
-module Recover
-  class CreateRecoverRequest
-    def self.call(user_id)
-      recovery_request = AccountRecoveryRequest.find_by(user_id: user_id)
```

Comment:
Should the associated model be removed?

https://github.com/18F/identity-idp/blob/stevegsa-rm-ial2-recovery/app/models/account_recovery_request.rb

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/app/models/user.rb#L46

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/spec/models/user_spec.rb#L15

And the table itself?

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/db/schema.rb#L18-L26

Reply:
>And the table itself?

Will we be able to follow-up to remove this? Not sure if it has to happen only after a deploy with code changes to avoid conflicts during deploy, since I'd expect the table's probably not actively used.